lib/logstash/inputs/stomp.rb

```diff
@@ -37,6 +37,12 @@ class LogStash::Inputs::Stomp < LogStash::Inputs::Base
   #Auto reconnect interval in seconds
   config :reconnect_interval, :validate => :number, :default => 30
 
+  # Enable TLS/SSL connection?
+  config :ssl, :validate => :boolean, :default => false
```

Comment:
I understnad if you aren't willing or interested in adding explicit certificate support. Maybe we can compromise.

Can you instead add documentation that tells users the necessary steps for creating a keystore with their trusted certificate authorities and also how to provide this file to Logstash, and also the caveats (it overrides the global keystore used by the whole Java process)?